Week1/homework/app.js

```diff
@@ -1,11 +1,122 @@
-'use strict';
 
```

Comment:
Looks like an extra space here. In fact, I think there looks like to be lots of strange spacing. Can you run prettier on your files so they fix the formatting issues?